packages/telemetry/karma.conf.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const karmaBase = require('../../config/karma.base');
+
+const files = [`src/**/*.test.ts`];
+
+module.exports = function (config) {
+  const karmaConfig = {
+    ...karmaBase,
+    // files to load into karma
+    files,
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
```

Comment:
nit: perhaps we could omit these comments in `karmaConfig` (not sure if they are just part of the template or wanted to be left in)